### PR TITLE
Gearbox 

### PR DESCRIPTION
**Pull request by qshao-gauntlet**
## What?
ie: Add support for uniswap contract events 

## How? 
ie: I used [abi-parser](https://nansen-contract-parser-prod.web.app/) or cli tools to build the table definitions

## Related PRs (optional)
PRs that are related to this or may need to be deployed before this PR

### CODE REVIEW

File: dags/resources/stages/parse/table_definitions/gearbox/DegenNFTV3_event_NewCreditFacadeAdded.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+    "parser": {
+        "abi": {
+            "anonymous": false,
+            "inputs": [
+                {
+                    "indexed": true,
+                    "internalType": "address",
+                    "name": "",
```
Comment (kome12) [inline dags/resources/stages/parse/table_definitions/gearbox/DegenNFTV3_event_NewCreditFacadeAdded.json:9]: Will need to add a name for this field.

File: dags/resources/stages/parse/table_definitions/gearbox/DegenNFTV3_event_NewCreditFacadeAdded.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        "contract_address": "0xb829a5b349b01fc71afe46e50dd6ec0222a6e599",
+        "field_mapping": {},
+        "type": "log"
+    },
+    "table": {
+        "dataset_name": "gearbox",
+        "schema": [
+            {
+                "description": "",
+                "name": "",
```
Comment (kome12) [inline dags/resources/stages/parse/table_definitions/gearbox/DegenNFTV3_event_NewCreditFacadeAdded.json:25]: Will need to add a name for this field.

File: dags/resources/stages/parse/table_definitions/gearbox/DegenNFTV3_event_NewCreditFacadeRemoved.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
+{
+    "parser": {
+        "abi": {
+            "anonymous": false,
+            "inputs": [
+                {
+                    "indexed": true,
+                    "internalType": "address",
+                    "name": "",
```
Comment (kome12) [inline dags/resources/stages/parse/table_definitions/gearbox/DegenNFTV3_event_NewCreditFacadeRemoved.json:9]: Will need to add a name for this field.

File: dags/resources/stages/parse/table_definitions/gearbox/DegenNFTV3_event_NewCreditFacadeRemoved.json [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+        "contract_address": "0xb829a5b349b01fc71afe46e50dd6ec0222a6e599",
+        "field_mapping": {},
+        "type": "log"
+    },
+    "table": {
+        "dataset_name": "gearbox",
+        "schema": [
+            {
+                "description": "",
+                "name": "",
```
Comment (kome12) [inline dags/resources/stages/parse/table_definitions/gearbox/DegenNFTV3_event_NewCreditFacadeRemoved.json:25]: Will need to add a name for this field.

File: dags/resources/stages/parse/table_definitions/gearbox/DegenNFTV3_event_NewMinterSet.json [hunk 5]
```diff
@@ -0,0 +1,32 @@
+{
+    "parser": {
+        "abi": {
+            "anonymous": false,
+            "inputs": [
+                {
+                    "indexed": true,
+                    "internalType": "address",
+                    "name": "",
```
Comment (kome12) [inline dags/resources/stages/parse/table_definitions/gearbox/DegenNFTV3_event_NewMinterSet.json:9]: Will need to add a name for this field.

File: dags/resources/stages/parse/table_definitions/gearbox/DegenNFTV3_event_NewMinterSet.json [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+        "contract_address": "0xb829a5b349b01fc71afe46e50dd6ec0222a6e599",
+        "field_mapping": {},
+        "type": "log"
+    },
+    "table": {
+        "dataset_name": "gearbox",
+        "schema": [
+            {
+                "description": "",
+                "name": "",
```
Comment (kome12) [inline dags/resources/stages/parse/table_definitions/gearbox/DegenNFTV3_event_NewMinterSet.json:25]: Will need to add a name for this field.

**Review by kome12 — CHANGES_REQUESTED** [review]
The CI/CD is currently failing because some of the name field has an empty string. Can you please add the name field?

File: dags/resources/stages/parse/table_definitions/gearbox/DegenNFTV3_event_NewCreditFacadeAdded.json (hunk 1 above)
Comment (zigzag2002) [reply dags/resources/stages/parse/table_definitions/gearbox/DegenNFTV3_event_NewCreditFacadeAdded.json]: fixed these

**Review by zigzag2002 — APPROVED** [review]

**Comment by zigzag2002** [comment]
> The CI/CD is currently failing because some of the name field has an empty string. Can you please add the name field?

Added names for all events, please approve when you get the chance to look over the changes

**Review by kome12 — APPROVED** [review]

**Pull request merged**